### DISCUSSION TITLE
##八分街离奇出现空洞，空洞调查协会人员公布调查初期结果（九分街最新新闻公布社克莱因报社为您报道）

### DISCUSSION THREAD

**Discussion by EllenJoe14** [share121/inter-knot#284] [其他]
八分街出现不明空洞，目前仍未得到其生成的准确答复。

起初，接到街区居民的建诉，称八分街西南郊区的远方海洋，一个庞然大物正悄悄滋生着，从中出现大量不明以骸生物。治安局接取随即前往调查。

治安局赶到现场，将部分以骸清除过后，开始在现场采样调查。

不久几天后，西南郊区一座古城堡的居住者瓦尔登一家族报案称：前几天治安局前去清扫以骸的地方，现如今已出现一个空洞。家族的大小姐希佩露与现场驻守的治安官说明情况。

现空洞调查协会正着手调查中。

一名治安官称：各位居民切莫慌张，经多方势力合作研究，空洞在此很长一段时间内暂时不会膨胀。涵请各位谅解。现在，我们将封锁西南郊区至莫海的路线，并派人驻守。

调查初期结果阐述：空洞大概率是由于海底不明以骸所产生的以太活性波次过大，导致的涌能松动，而所产生的。

空洞的侵蚀效果常人是难以忍受的，且若你不持有【萝卜】，你将无法轻易返回。近些年来，空洞波次日趋增大，空洞内活性逐渐增大，这对普通居民而言无疑是坏消息，盗洞客团队也较为猖獗。

望居民们妥善出行，避免踏入禁止通行的涉洞场地。

稍后的消息与结果，本社将持续为您报道。

新闻版号：54628676921740375
投稿网址：114-514-1919@zzz.com
记者：阿迷·诺斯

**Comment by Zhishirur** [comment]
嗯呢!嗯呢嗯呢!(已阅)

**Comment by lllliufeng** [comment]
快点解决吧，我在隔壁街区瘆得慌。。
